### DISCUSSION TITLE
Peculiar Pattern for Scatter y-score x-q_value

### DISCUSSION THREAD

**Discussion by animesh** [lazear/sage#149] [Q&A] [answered]
I see this peculiar pattern for scatter between score and q_value
![image](https://github.com/user-attachments/assets/d315b9d9-c59c-42d7-9a88-187b4f035da4)
wondering what could be the reason for this? Any reference would be appreciated 💯

**Comment by lazear** [comment, marked as answer]
Thanks for the report. This a bug which I've just fixed - next time please open this as an issue (which you clearly know how to do :smile:)
